src/pages/ActiveTasks/ActiveTasks.tsx

```diff
@@ -1,23 +1,67 @@
-import { useState, ChangeEvent, useEffect } from "react";
+import SearchIcon from "@mui/icons-material/Search";
+import { InputAdornment } from "@mui/material";
+import TextField from "@mui/material/TextField";
+import {
+  doc,
+  getDoc,
+  getFirestore,
+  setDoc,
+  updateDoc,
+} from "firebase/firestore";
+import { ChangeEvent, useEffect, useState } from "react";
 import ActiveTaskTile from "../../components/ActiveTaskTile/ActiveTaskTile";
+import Header from "../../components/Header/Header";
 import Navigation from "../../components/Navigation/Navigation";
+import Popup from "../../components/Popup/Popup";
+import { app } from "../../firebase";
 import { ActiveTask } from "../../types/Task";
 import "./ActiveTasks.scss";
-import Header from "../../components/Header/Header";
-import TextField from "@mui/material/TextField";
-import { InputAdornment } from "@mui/material";
-import SearchIcon from "@mui/icons-material/Search";
+import { useNavigate } from "react-router-dom";
 import { useFirestore } from "../../hooks/useFireStore";
+import dayjs from "dayjs";
 
 type ActiveTasksItem = {
   [key: string]: boolean;
 };
 
+const db = getFirestore(app);
+
+type UserData = {
+  bio: string;
+  email: string;
+  id: string;
+  img: string;
+  name: string;
+  totalScore: number;
+};
+
```

Comment:
These types are defined in the types folder so you should be able to import them in and use them